node/time_heap.js

```diff
@@ -77,6 +77,7 @@ function TimeHeap(options) {
     self.lastTime = self.timers.now();
     self.timer = null;
     self.end = 0;
+    self.lastRun = 0;
```

Comment:
Would you buy -Infinity? It’s legit, but not a smallint. I won’t insist. The 70’s are not likely to occur again and might as well be infinity ago.